### PR TITLE
chore: Toolbar improvements

### PR DESCRIPTION
**Pull request by chpalac**
# Overview

This PR adds some small improvements for `Toolbar` such has removing `ToolbarRadio` from toolbar exports, sets `ToolbarButton` default appearance and export `Toolbar` component.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a074e18ba5559606fd2a596bc7643cddf78e3678'>a074e18ba5559606fd2a596bc7643cddf78e3678</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/Avatar-mount_base.html">906</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/Avatar-mount.html">906</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/Button-mount_base.html">561</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/Button-mount.html">569</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/FluentProvider-mount_base.html">1791</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/FluentProvider-mount.html">1850</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/FluentProviderWithTheme-mount_base.html">303</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/FluentProviderWithTheme-mount.html">291</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">241</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">231</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">343</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">327</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/MakeStyles-mount_base.html">1572</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23346/perf-test-react-components/MakeStyles-mount.html">1590</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Review by ling1726 — APPROVED** [review]

**Review by spmonahan — APPROVED** [review]

File: packages/react-components/react-components/src/unstable/index.ts [hunk 1]
```diff
@@ -71,6 +71,31 @@ export type {
...
 
+export {
+  Toolbar,
+  renderToolbar_unstable,
+  toolbarClassNames,
+  useToolbarStyles_unstable,
+  useToolbar_unstable,
+  ToolbarDivider,
+  useToolbarDividerStyles_unstable,
+  ToolbarToggleButton,
```
Comment (khmakoto) [inline packages/react-components/react-components/src/unstable/index.ts:82]: _nit:_ Ordering

```suggestion
  Toolbar,
  ToolbarDivider,
  ToolbarToggleButton,
  renderToolbar_unstable,
  toolbarClassNames,
  useToolbar_unstable,
  useToolbarDividerStyles_unstable,
  useToolbarStyles_unstable,
```

File: packages/react-components/react-components/src/unstable/index.ts [hunk 2]
```diff
@@ -71,6 +71,31 @@ export type {
...
+  ToolbarContextValues,
+  ToolbarProps,
+  ToolbarSlots,
+  ToolbarState,
+  ToolbarButtonProps,
+  ToolbarButtonState,
+  ToolbarDividerProps,
+  ToolbarDividerState,
+  ToolbarToggleButtonProps,
+  ToolbarToggleButtonState,
```
Comment (khmakoto) [inline packages/react-components/react-components/src/unstable/index.ts:96]: _nit:_ Ordering

```suggestion
  ToolbarButtonProps,
  ToolbarButtonState,
  ToolbarContextValue,
  ToolbarContextValues,
  ToolbarDividerProps,
  ToolbarDividerState,
  ToolbarProps,
  ToolbarSlots,
  ToolbarState,
  ToolbarToggleButtonProps,
  ToolbarToggleButtonState,
```

**Review by khmakoto — APPROVED** [review]

**Pull request merged**